docs/src/modules/components/ApiPage.js

```diff
@@ -229,7 +229,7 @@ function ApiDocs(props) {
     ...componentDescriptionToc,
     componentStyles.name && createTocEntry('component-name'),
     createTocEntry('props'),
-    componentStyles.classes && createTocEntry('css'),
+    componentStyles.classes?.length > 0 && createTocEntry('css'),
```

Comment:
Why is `.classes` defined if it's empty?

Reply:
> Why is `.classes` defined if it's empty?

You mean empty array right? Not sure. Maybe to be able to loop over them and avoid null checks later everytime whether it is null or an array. I think it comes from here,
https://github.com/mui-org/material-ui/blob/cd247a197b772b74d81db70e1c3dc5d0a019c447/docs/src/modules/utils/parseStyles.ts#L16-L23

Reply:
Good point. I agree, sometimes it hides bugs. I committed this change https://github.com/mui-org/material-ui/pull/29622#discussion_r747367374 without thinking. Reverted it now.